docs/documentation/upgrading/topics/changes/changes-26_3_0.adoc

```diff
@@ -24,6 +24,18 @@ When you want the user, who is authenticated to your client application, to link
 mechanism with the action `idp_link`. The proprietary custom protocol for client initiated account linking is deprecated now and might be removed in the future versions. For more information, see the
 Client initiated account link section of the link:{developerguide_link}[{developerguide_name}].
 
+=== Deprecation of `spi-connections-infinispan-quarkus-site-name`
+
+The option `spi-connections-infinispan-quarkus-site-name` is deprecated, and it will be removed in the future.
+Use `spi-cache-embedded-default-site-name` instead.
+Specifying the site name is no longer required
```

Comment:
The statement "Specifying the site name is no longer required" looks incomplete to me. The docs still say that you specify that option when using multi-site HA. Which one is true? Please clarify. Again, this can be a follow-up unless it needs any code changes.